### PR TITLE
Insurance pool positions

### PR DESCRIPTION
**Pull request by stevenyyan**
### Motivation

Display current deposits for the insurance portfolio pool

### Changes

Add pool positions

### CODE REVIEW

File: components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx [hunk 1]
```diff
@@ -32,26 +34,32 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight }: ADProps) => {
...
                                 onMouseLeave={() => setShowButton(-1)}
                             >
-                                <TableCell>{tracer.market}</TableCell>
-                                <TableCell>{tracer.realisedAPY}</TableCell>
-                                <TableCell>{tracer.ownership}</TableCell>
-                                <TableCell>{tracer.unrealisedValue}</TableCell>
+                                <TableCell>{insuranceContract.market}</TableCell>
+                                <TableCell>{toPercent(insuranceContract.apy.toNumber())}</TableCell>
+                                <TableCell>-</TableCell>
+                                <TableCell>-</TableCell>
```
Comment (dospore) [inline components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx]: You could add the users ownership here

Comment (dospore) [reply components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx]: see insurance page for an example

Comment (stevenyyan) [reply components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx]: updated

File: components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx [hunk 2]
```diff
@@ -32,26 +34,32 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight }: ADProps) => {
...
-                                <TableCell>{tracer.ownership}</TableCell>
-                                <TableCell>{tracer.unrealisedValue}</TableCell>
+                                <TableCell>{insuranceContract.market}</TableCell>
+                                <TableCell>{toPercent(insuranceContract.apy.toNumber())}</TableCell>
+                                <TableCell>{insuranceContract.liquidity.toNumber()}</TableCell>
+                                <TableCell>-</TableCell>
                                 <TableCell>
                                     <div className="flex">
-                                        {tracer.instant} <Button className={show ? 'ml-5' : 'hide'}>Withdraw</Button>
+                                        -{' '}
```
Comment (dospore) [inline components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx:51]: is this meant to be a minus sign?

File: components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx [hunk 3]
```diff
@@ -32,26 +34,32 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight }: ADProps) => {
...
                                 onMouseEnter={() => setShowButton(i)}
                                 onMouseLeave={() => setShowButton(-1)}
                             >
-                                <TableCell>{tracer.market}</TableCell>
-                                <TableCell>{tracer.realisedAPY}</TableCell>
-                                <TableCell>{tracer.ownership}</TableCell>
-                                <TableCell>{tracer.unrealisedValue}</TableCell>
+                                <TableCell>{insuranceContract.market}</TableCell>
+                                <TableCell>{toPercent(insuranceContract.apy.toNumber())}</TableCell>
+                                <TableCell>{insuranceContract.liquidity.toNumber()}</TableCell>
```
Comment (dospore) [inline components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx]: If this is the users balance I think there is a userBalance key

**Review by dospore — COMMENTED** [review]
Just a few extra things

File: components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx (hunk 2 above)
Comment (stevenyyan) [reply components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx:51]: a placeholder for TBD data field

File: components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx (hunk 3 above)
Comment (stevenyyan) [reply components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx]: sure weird but userBalance is always empty while liquidity gets the balance amount

**Review by dospore — APPROVED** [review]

**Pull request merged**